docs/root/start/building.rst

```diff
@@ -6,15 +6,15 @@ Building
 
 The Envoy build system uses `Bazel <https://bazel.build/>`_.
 
-In order to ease initial building and for a quick start, we provide an Ubuntu 16 based docker container that
-has everything needed inside of it to build and *statically link* Envoy, see :repo:`ci/README.md`.
+In order to ease initial building and for a quick start, we provide an Ubuntu 16 and a Windows based docker containers
+that have everything needed inside of it to build and *statically link* Envoy, see :repo:`ci/README.md`.
 
 In order to build without using the Docker container, follow the instructions at :repo:`bazel/README.md`.
 
 .. _install_requirements:
 
-Requirements
-------------
+Linux/Mac Requirements
+----------------------
```

Comment:
theres still something confusing about this

iiuc this is building for a linux/mac target - but could also be on windows